images/base/files/usr/local/bin/entrypoint

```diff
@@ -50,12 +130,16 @@ fix_mount() {
     sync
   fi
 
-  echo 'INFO: remounting /sys read-only'
-  # systemd-in-a-container should have read only /sys
-  # https://systemd.io/CONTAINER_INTERFACE/
-  # however, we need other things from `docker run --privileged` ...
-  # and this flag also happens to make /sys rw, amongst other things
-  mount -o remount,ro /sys
+  if [[ -z "${userns}" ]]; then
+    echo 'INFO: remounting /sys read-only'
+    # systemd-in-a-container should have read only /sys
+    # https://systemd.io/CONTAINER_INTERFACE/
+    # however, we need other things from `docker run --privileged` ...
+    # and this flag also happens to make /sys rw, amongst other things
+    #
+    # This step is skipped when running inside UserNS, because it fails with EACCES.
```

Comment:
we need to disable udev

Reply:
What happens if not?

Reply:
udev is automatically disabled by /sys being `ro`, which is one of the systemd-in-a-container requirements.
when udev is not disabled, sometimes startup is broken by trying to "start" devices which cannot be started.
e.g. from the docker for mac xhyve.

Reply:
https://github.com/kubernetes-sigs/kind/issues/1474#issuecomment-613278644

Reply:
updated